streaming/src/main/scala/org/apache/spark/streaming/dstream/DStream.scala

```diff
@@ -906,12 +908,16 @@ abstract class DStream[T: ClassTag] (
   /**
    * Save each RDD in this DStream as at text file, using string representation
    * of elements. The file name at each batch interval is generated based on
-   * `prefix` and `suffix`: "prefix-TIME_IN_MS.suffix".
+   * `prefix` and `suffix`: "prefix-TIME_IN_MS.suffix". If the `CompressionCodec`
+   * is defined, it will use specific `CompressionCodec` to compress the text.
    */
-  def saveAsTextFiles(prefix: String, suffix: String = ""): Unit = ssc.withScope {
```

Comment:
@srowen , it seems Mima doesn't complain about this binary changing :). Yes, actually most of DStream API can be done by `transform()` with related RDD API, not sure which way is a better way to expose to user? What is your suggestion @tdas.